akanda/router/drivers/iptables.py

```diff
@@ -301,11 +309,6 @@ def _build_floating_ips(self, config):
             # people create these accidentally, just ignore them (because
             # iptables will barf if it encounters them)
             if fip.fixed_ip.version == fip.floating_ip.version:
-                rules.append(
```

Comment:
This only creates the rule for fixed IPs that are associated with some floater.  This rule needs to exist for the entire v4 cidr so that VMs *without* Floating IPs have proper SNAT.